### PR TITLE
Add outputs for VPN gateway and route table

### PR DESCRIPTION
**Pull request by hhoover**
This commit also has README updates for said outputs

### CODE REVIEW

File: outputs.tf [hunk 1]
```diff
@@ -56,3 +56,13 @@ output "public_subnet_ids" {
   description = "The IDs of the main public subnets that have been created."
   value       = module.vpc.public_subnets
 }
+
+output "main_route_table_id" {
+  description = "The IDs of the created VPN gateway in the VPC."
```
Comment (bmcustodio) [inline outputs.tf:61]:
```suggestion
  description = "The ID of the main route table that has been created."
```

File: outputs.tf [hunk 2]
```diff
@@ -56,3 +56,13 @@ output "public_subnet_ids" {
...
+  value       = module.vpc.vpc_main_route_table_id
+}
+
+output "vpn_gw_id" {
+  description = "The IDs of the created VPN gateway in the VPC."
```
Comment (bmcustodio) [inline outputs.tf:66]:
```suggestion
  description = "The ID of the VPN gateway that has been created."
```

**Review by bmcustodio — CHANGES_REQUESTED** [review]
Couple nits (and you'll likely have to re-run `make docs`).

**Comment by hhoover** [comment]
resolved, please re-review 😄

**Comment by hhoover** [comment]
It would be nice, but looks like I need to update / redo the PR. _sets reminder_

**Comment by hhoover** [comment]
@bmcustodio done with what's different, please review

**Review by PhilipSchmid — APPROVED** [review]

**Pull request merged**